api/src/test/java/uk/gov/dhsc/htbhf/dwp/service/HouseholdVerifierLegacyHouseholdTest.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.dhsc.htbhf.dwp.service;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.dwp.entity.legacy.LegacyAdult;
+import uk.gov.dhsc.htbhf.dwp.entity.legacy.LegacyHousehold;
+import uk.gov.dhsc.htbhf.dwp.model.AddressDTO;
+import uk.gov.dhsc.htbhf.dwp.model.PersonDTO;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.dhsc.htbhf.dwp.entity.LegacyHouseholdFactory.aHouseholdWithNoAdultsOrChildren;
+
+public class HouseholdVerifierLegacyHouseholdTest {
+
+    private final HouseholdVerifier householdVerifier = new HouseholdVerifier();
+    private final AddressDTO address = AddressDTO.builder()
```

Comment:
I think all the test data objects you create for this test should be in test data factories.

Reply:
Done.